src/modules/extras/components/Extras.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import * as React from 'react'
+import TileModel from '../../common/models/TileModel'
+import { ExtraModel } from '@integreat-app/integreat-api-client'
+import Tiles from '../../common/components/Tiles'
+import type { TFunction } from 'react-i18next'
+import {
+  SPRUNGBRETT_EXTRA,
+  SPRUNGBRETT_ROUTE,
+  WOHNEN_EXTRA, WOHNEN_ROUTE
+} from '../ExtrasConfig'
+import { ScrollView } from 'react-native'
+
+type PropsType = {|
+  extras: Array<ExtraModel>,
+  navigateToExtras: (path: string, isExternalUrl: boolean) => void,
+  t: TFunction
+|}
+
+export default class Extras extends React.Component<PropsType> {
```

Comment:
Rename to `ExtraTiles` or sth like that. Or is this a convention you agreed on?

Reply:
We also have `Categories`, so I would just leave it like that.